src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {NavLink} from 'react-router-dom';
+
+const Header = () => (
+  <header>
+    <h1>School App</h1>
+    <NavLink to="/" activeClassName="is-active" exact={true}>Activity Plan</NavLink>
+    <NavLink to="/create" activeClassName="is-active">Add Activity</NavLink>
```

Comment:
routes to constants. 
you can do:
`<NavLink exact... />` not `exact={true}` as it has bool type.